boards/others/esp32c3_supermini/doc/index.rst

```diff
@@ -0,0 +1,282 @@
+.. _esp32c3_supermini:
+
+ESP32-C3-SUPERMINI
+####################
+
+Overview
+********
+
+ESP32-C3-SUPERMINI is based on the ESP32-C3, a single-core Wi-Fi and Bluetooth 5 (LE) microcontroller SoC,
+based on the open-source RISC-V architecture. This board also includes a Type-C USB Serial/JTAG port.
+For more information, check `ESP32-C3-SUPERMINI`_
+
+Hardware
+********
+
+SoC Features:
+
+- IEEE 802.11 b/g/n-compliant
+- Bluetooth 5, Bluetooth mesh
+- 32-bit RISC-V single-core processor, up to 160MHz
+- 384 KB ROM
+- 400 KB SRAM (16 KB for cache)
+- 8 KB SRAM in RTC
+- 22 x programmable GPIOs
+- 3 x SPI
+- 2 x UART
+- 1 x I2C
+- 1 x I2S
+- 2 x 54-bit general-purpose timers
+- 3 x watchdog timers
+- 1 x 52-bit system timer
+- Remote Control Peripheral (RMT)
+- LED PWM controller (LEDC)
+- Full-speed USB Serial/JTAG controller
+- General DMA controller (GDMA)
+- 1 x TWAI®
+- 2 x 12-bit SAR ADCs, up to 6 channels
+- 1 x soc core temperature sensor
+
+For more information on the ESP32-C3 SOC, check the datasheet at `ESP32-C3 Datasheet`_ or the technical reference
+manual at `ESP32-C3 Technical Reference Manual`_.
+
+Supported Features
+==================
+
+Current Zephyr's ESP32-C3-SUPERMINI board supports the following features:
```

Comment:
put board name as used with west build here with \`\` around it and use board target instead of board